protocol/src/protocol.ts

```diff
@@ -639,7 +639,7 @@ export interface ImplementationResult extends V {
 	/**
 	 * The ranges and locations belong to the implementation result.
 	 */
-	result?: (RangeId | lsp.Location)[];
+	results?: (RangeId | lsp.Location)[];

```

Comment:
@dbaeumer plural is more appropriate for this property since we are talking about an array. I believe this was missed by the fix for issue #26. I went ahead and made the changes necessary to the protocol.

Reply:
@jumattos I actually kept the singular even if it is an array to make all results consistent. See `DeclarationResult`, `DefinitionResult` or `HoverResult`. I only used plural if it is a specific property. So I would leave the property `result` sigular.